Source/FieldSolver/SpectralSolver/SpectralSolver.H

```diff
@@ -76,13 +76,15 @@ class SpectralSolver
                         const amrex::Array<amrex::Real,3>& v_comoving,
                         const amrex::RealVect dx,
                         const amrex::Real dt,
-                        const bool pml = false,
-                        const bool periodic_single_box = false,
-                        const bool update_with_rho = false,
-                        const bool fft_do_time_averaging = false,
-                        const bool J_linear_in_time = false,
-                        const bool dive_cleaning = false,
-                        const bool divb_cleaning = false);
+                        const bool pml,
+                        const bool periodic_single_box,
+                        const bool update_with_rho,
+                        const bool fft_do_time_averaging,
+                        const bool J_linear_in_time,
+                        const bool dive_cleaning,
+                        const bool divb_cleaning,
+                        const bool pml_dive_cleaning,
+                        const bool pml_divb_cleaning);
```

Comment:
Here I removed the defaults, because in my opinion it is safer to pass all arguments, when there are so many, to avoid confusing some of the ones that are similar (e.g. passing `true` for `dive_cleaning` instead of `pml_dive_cleaning` and not realizing that because the compiler does not complain about missing arguments or things like this).